tst/EnergyPlus/unit/InputProcessor.unit.cc

```diff
@@ -833,7 +868,10 @@ TEST_F(InputProcessorFixture, parse_idf_extensible_blank_extensibles)
                          {"temperature_convergence_tolerance_value", 0.4000},
                          {"solar_distribution", "FullExterior"},
                          {"maximum_number_of_warmup_days", 25},
-                         {"minimum_number_of_warmup_days", 6}}}}}};
+                         {"minimum_number_of_warmup_days", 6}}}}},
+                     {"Version",
+                      {{"",
+                        {{"version_identifier", "9.4"}}}}}};
```

Comment:
I don't mind the removal of version strings from unit tests where they aren't needed.  But what happened to make these need to be here?  The change_version script won't find these, so that'll either need to be modified or we'll just need to update these manually.

Reply:
Same reason as why `Building` and `GlobalGeometryRules` are added to all tests there: because if not present, they are automatically added by `process_idf`. So by adding them explicitly, you can better control the way the roundtrip is done. 

You bring a good point though: I will bump to 9.5 locally and see if there's a need to update these or not (it may throw a warning for expected version...), in which case I'll use `DataStringGlobals::MatchVersion` to avoid the problem.

**Edit**: After bumping to 9.5, I did use DataStringGlobals

Reply:
`Building` and `GlobalGeometryRules` are automatically added if they are not present because they are the only two required objects in all of E+. They must be present for an IDF to be valid. `Version` is not automatically added by `process_idf` but it does check to see if version numbers match during `process_idf`. Why does `Version` need to be added here? I personally don't mind since it makes it more explicit and unit tests will fail if they are not updated properly if something changes.

Reply:
Because I made `process_idf` add Version automatically in this PR. The reasoning is that otherwise you automatically get a warning:

```
** Warning** Version: missing in IDF, processing for EnergyPlus version=\"" + MatchVersion + "\""
```

which is annoying where you are calling `compare_err_stream` later on, and the major reason why a bunch of unit tests had an explicit `Version` object in there, which required us to bump the version in these units tests when bumping the `cmake/Version.cmake`.

I'm not sure I'm really clearly stating the above, hopefully I am?